docs/inference-providers/_toctree.yml

```diff
@@ -6,11 +6,22 @@
     title: Pricing and Billing
   - local: hub-integration
     title: Hub integration
-  - local: register-as-a-provider
-    title: Register as an Inference Provider
   - local: security
     title: Security
 
+- title: Guides
+  sections:
+  - local: guides/first-api-call
+    title: Your First API Call
+  - local: guides/building-first-app
+    title: Building Your First AI App
+  - local: guides/structured-output
+    title: Structured Outputs with LLMs
+  - local: guides/function-calling
+    title: Function Calling
+  - local: guides/gpt-oss
+    title: How to use OpenAI gpt-oss
+
```

Comment:
Would also put the task guides (like "chat completion", "text-to-image") below the guides, rather than below the individual providers.